elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ProviderDefinition.java

```diff
@@ -28,6 +30,7 @@
  */
 class ProviderDefinition extends SimpleResourceDefinition {
 
+    static final ResourceRegistration PATH = ResourceRegistration.of(PathElement.pathElement(ElytronOidcDescriptionConstants.PROVIDER), Stability.DEFAULT);
```

Comment:
Just to check, where does this get used?

Reply:
Here: https://github.com/wildfly/wildfly/blob/1983491071bff113d53a789292cb2389bb7d692f/elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcSubsystemSchema.java#L100